GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -467,7 +467,7 @@ private bool HandlePushOnExit(ref bool isError, FormProcess form)
                 if (AppSettings.AutoPullOnPushRejectedAction == null)
                 {
                     bool cancel = false;
-                    string destination = PushToRemote.Checked ? _NO_TRANSLATE_Remotes.Text : PushDestination.Text;
```

Comment:
While the rewrite is OK, is this really what is desired?

Reply:
Yes.